servant-examples/auth-combinator/auth-combinator.hs

```diff
@@ -31,12 +32,12 @@ instance HasServer rest => HasServer (AuthProtected :> rest) where
   route Proxy a = WithRequest $ \ request ->
     route (Proxy :: Proxy rest) $ do
       case lookup "Cookie" (requestHeaders request) of
-        Nothing -> return $ failWith $ HttpError status401 (Just "Missing auth header.")
+        Nothing -> return $! FailFatal err401 { errBody = "Missing auth header" }
```

Comment:
No real reason - at some point in the writing of this there was a function instead of a constructor on the RHS and the `$!` seemed to make sense in that context, but not anymore. I'll remove